src/data/polynomial/coeff.lean

```diff
@@ -134,6 +134,11 @@ begin
   { exact λ h1, (h1 (nat.mem_antidiagonal.2 rfl)).elim }
 end
 
+theorem coeff_X_pow_mul (p : polynomial R) (n d : ℕ) :
+  coeff (polynomial.X ^ n * p) (d + n) = coeff p d :=
```

Comment:
```suggestion
  coeff (X ^ n * p) (d + n) = coeff p d :=
```

Reply:
Oh, I guess this matches the one above, just not the one below

Reply:
Yeah these things also slightly annoyed me about the original versions, also the not using dot notation for coeff (which is possibly part of the reason I couldn't find this lemma when I looked for it before), but I figured I'd rather not keep changing random lines in ways that don't really make a difference. It will show up with the polynomial prefix in doc_gen anyway.